cli/src/main/java/org/jboss/as/cli/impl/aesh/CLICompletionHandler.java

```diff
@@ -63,6 +65,20 @@ public void complete(AeshCompleteOperation co) {
 
         LOG.debugf("Completing {0}", co.getBuffer());
         cliCompleter.complete(ctx, co, this);
+
+        if (ctx.getConfig().isColorOutput()) {
+            List<TerminalString> completionCandidates = co.getCompletionCandidates();
+            List<TerminalString> requiredCandidates = new ArrayList<>();
+            for (TerminalString candidate : completionCandidates) {
+                if (candidate.toString().endsWith("*") && !"*".equals(candidate.toString())) {
+                    TerminalString newCandidate = Util.formatRequired(candidate);
+                    requiredCandidates.add(newCandidate);
+                }
+            }
+            completionCandidates.removeIf(candidate -> candidate.toString().endsWith("*"));
```

Comment:
we try to avoid usage of lambda, sorry ;-)

Reply:
I know, I just thought that since the CLI is not a long running process it was OK.